### PR TITLE
hello-microverse

### PR DESCRIPTION
**Pull request by Zeph180**
- Create hello-microverse project
- Setup a linter to catch syntax errors
- Add .CSS file to store styles for the markup
- Add readme file to document project 
- Add a test.md file to gitignore
-Add package.json to store dependencies

### CODE REVIEW

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Zeph180, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Descriptive commit history ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
- [ ] [REQUIRED] Good job so far! Please add more information about this milestone to your `Pull Request` Description because `Pull Request` should have a descriptive `title` telling you what changes are inside and `summary` describing introduced changes in `more detail`🙏 

image

- [ ] [REQUIRED] Please implement my suggestions! You forgot to install `Linter Actions`, please install them [based on the list](https://github.com/microverseinc/linters-config/tree/master/html-css) provided by `Microverse` 🙏 

image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Zeph180** [comment]
Are all linter actions suppossed to pass?

**Comment by Zeph180** [comment]
I am failing one

**Comment by DuaneDave** [comment]
Hey @Zeph180 👋 according to the requirements, all linters tests are supposed to pass to reduce the chances of bugs in your code.

**Comment by Zeph180** [comment]
I finally passed all the tests and requested a review

File: README.md [hunk 1]
```diff
@@ -37,249 +37,73 @@ After you're finished please remove all the comments and instructions!
...
-  - [Answer_2]
+I would like to thank microverse for the great fullstack programme
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (DuaneDave) [inline README.md]: [OPTIONAL] I have noticed that this does not link to the actual license, instead I get a 404 error when I click it.



Kindly add the correct file path which is `./MIT`.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Zeph180 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Quality efforts made
✔️ Meaningful commits
✔️ Linters are passing



 ## Required Changes ♻️
- [REQUIRED] As requested by the previous reviewer, Kindly add a descriptive PR heading and a summary stating the things achieved in this PR
pr head

_**Kindly click on the three dots by the top right corner to edit this**_

## Optional suggestions
- When writing a commit message, I suggest using capital letters for the first words to make them more professional.
read also the comments below.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Zeph180** [comment]
@DuaneDave is that description ok

**Comment by mukhammadrizooff** [comment]
> @DuaneDave is that description ok

### No bro, please add more information about what did you do in this milestone, your current description is so simple and short 😃 

image

**Comment by DuaneDave** [comment]
No it's not.

I suggest you add a summary of all you have done e.g

- Set up linters to catch syntax errors
- Add a CSS file to hold the styles
- Add an HTML file for markup
- Document project in readme.md file
- Add a test.md file in gitignore.

This will suffice 🤗

**Comment by Zeph180** [comment]
I am humbled  @DuaneDave sure this is a great way to learn. Thanks alot

**Comment by DuaneDave** [comment]
@Zeph180 its okay 🤗.

I'm sure you'd master all these things shortly.

Good luck 🤝

**Comment by Zeph180** [comment]
@DuaneDave 
I have also added the link to the MIT license.

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Zeph180 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**